Makefile

```diff
@@ -1,94 +0,0 @@
-VIRTUALENV = virtualenv
```

Comment:
We still need a Makefile. See https://github.com/mozilla-services/autograph/blob/master/Makefile for examples